packages/features/ee/managed-event-types/lib/handleChildrenEventTypes.ts

```diff
@@ -159,6 +164,10 @@ export default async function handleChildrenEventTypes({
       ? { delete: true }
       : undefined;
   };
+  await removePreviousSlug({
+    userId: [...(newUserIds ?? []), ...(oldUserIds ?? [])],
+    previousSlug: managedEventTypeValues.slug,
+  });
```

Comment:
Remove the previousSlugs for the `event-type` of users of current event for which `previousSlug === slug`